Python/Product/PythonTools/PythonTools/Intellisense/IntellisenseController.cs

```diff
@@ -716,67 +716,67 @@ private class ExpressionCompletionWalker : PythonWalker {
             // TODO: Take into account params arrays
             // TODO: need to parse and see if we have keyword arguments entered into the current signature yet
             PythonSignature sig = sigHelpSession.SelectedSignature as PythonSignature;
-            if (sig != null) {
-                var prevBuffer = sig.ApplicableToSpan.TextBuffer;
-                var textBuffer = _textView.TextBuffer;
+            if (sig == null) {
+                return;
+            }
 
-                var targetPt = _textView.BufferGraph.MapDownToFirstMatch(
-                    new SnapshotPoint(_textView.TextBuffer.CurrentSnapshot, position),
-                    PointTrackingMode.Positive,
-                    EditorExtensions.IsPythonContent,
-                    PositionAffinity.Successor
-                );
+            var prevBuffer = sig.ApplicableToSpan.TextBuffer;
+            var textBuffer = _textView.TextBuffer;
 
-                if (targetPt == null) {
-                    return;
-                }
-                var span = targetPt.Value.Snapshot.CreateTrackingSpan(targetPt.Value.Position, 0, SpanTrackingMode.EdgeInclusive);
+            var targetPt = _textView.BufferGraph.MapDownToFirstMatch(
+                new SnapshotPoint(_textView.TextBuffer.CurrentSnapshot, position),
```

Comment:
All I really did was unindent the code by one level, so there's not much new here. It looks like that variable was and always has been unused. But I'll switch to using it, as it makes sense.